index.js

```diff
@@ -133,15 +150,14 @@ module.exports = {
             }
         };
 
-        //Adding event listeners to take automatic screenshot
+        // Adding event listeners to take automatic screenshot
         if (autoCaptureOptions.indexOf('click') !== -1) {
             flow.on('scheduleTask', function (task) {
```

Comment:
How come I don't see [changes](https://github.com/paypal/nemo-screenshot/blob/develop/index.js#L140) in develop in your PR?